routers/install.go

```diff
@@ -81,7 +81,7 @@ func Install(ctx *context.Context) {
 	form.RepoRootPath = setting.RepoRootPath
 	form.LFSRootPath = setting.LFS.ContentPath
 
-	// Note(unknwon): it's hard for Windows users change a running user,
```

Comment:
This is probably not a typo. Remember that gogs was written by https://github.com/unknwon